src/main/java/teammates/ui/webapi/GetFeedbackSessionsAction.java

```diff
@@ -47,8 +51,13 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
             }
 
             if (courseId != null) {
-                CourseAttributes courseAttributes = logic.getCourse(courseId);
-                gateKeeper.verifyAccessible(logic.getStudentForGoogleId(courseId, userInfo.getId()), courseAttributes);
+                if (isCourseMigrated(courseId) && isAccountMigrated(userInfo.getId())) {
```

Comment:
Shouldn't be a need to check both. Can assume accounts are migrated before courses.